cfg/config.yaml

```diff
@@ -78,15 +78,26 @@ node:
     - kubernetes
 
   kubernetes:
-    defaultconf: /etc/kubernetes/config    
+    confs:
+      - "/var/lib/kubelet/kubeconfig"
+      - "/etc/kubernetes/config"
+    defaultconf: "/etc/kubernetes/config"
 
   kubelet:
     cafile:
       - "/etc/kubernetes/pki/ca.crt"
+      - "/etc/kubernetes/certs/ca.crt"
+    svc:
+      - "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
+      - "/etc/systemd/system/kubelet.service"
     bins:
       - "hyperkube kubelet"
       - "kubelet"
+    kubeconfig:
+      - "/var/lib/kubelet/kubeconfig"
+      - "/etc/kubernetes/kubelet.conf"
     confs:
+      - "/etc/default/kubelet"
```

Comment:
Similarly here, please add `/etc/default/kubelet` at the end of the list of confs. This would break an Ubuntu kubeadm install as there is a `/etc/default/kubelet` file as well as `/var/lib/kubelet/config.yaml`, and it's the latter that kube-bench should be using here